### PR TITLE
Added docker file and updated readme

### PR DESCRIPTION
**Pull request by SagarVishal**

### CODE REVIEW

File: hydrastart.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    # restart: always
+    # networks:
+    #   - hydraguide
+    # volumes:
+    #   - ./gitea:/data
+    #   - /etc/timezone:/etc/timezone:ro
+    #   - /etc/localtime:/etc/localtime:ro
+    # ports:
+    #   - "3000:3000"
+    #   - "2228:22"
```
Comment (ekta-kakadia) [inline hydrastart.yaml]: add extra line here

File: hydrastart-postgres.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  postgresd:
+    image: postgres:9.6
+    ports:
+      - "5432:5432"
+    environment:
+      - POSTGRES_USER=hydra
+      - POSTGRES_PASSWORD=secret
+      - POSTGRES_DB=hydra
+    networks:
+      - hydraguide
```
Comment (ekta-kakadia) [inline hydrastart-postgres.yaml]: here too extra line

File: web/.env.docker [hunk 3]
```diff
@@ -1 +1,2 @@
 HYDRA_PROXY_URL=http://localhost:4445
+PORT=9000
```
Comment (ekta-kakadia) [inline web/.env.docker]: here too

File: web/.env.example [hunk 4]
```diff
@@ -1 +1,2 @@
 HYDRA_PROXY_URL=http://localhost:4445
+PORT=9000
```
Comment (ekta-kakadia) [inline web/.env.example]: extra line here

Comment (SagarVishal) [reply web/.env.example]: done

File: web/.env.docker (hunk 3 above)
Comment (SagarVishal) [reply web/.env.docker]: done

File: hydrastart-postgres.yaml (hunk 2 above)
Comment (SagarVishal) [reply hydrastart-postgres.yaml]: done

File: hydrastart.yaml (hunk 1 above)
Comment (SagarVishal) [reply hydrastart.yaml]: done

**Review by ekta-kakadia — APPROVED** [review]

File: web/package.json [hunk 5]
```diff
@@ -44,6 +44,7 @@
     "husky": "^4.3.8",
     "jest": "^26.6.3",
     "lint-staged": "^10.5.3",
+    "moment": "^2.29.1",
```
Comment (tapanbavaliya) [inline web/package.json:47]: @Vishalsag is moment in use, because not seen use in this PR.

Comment (SagarVishal) [reply web/package.json:47]: yes it is used to convert date

**Review by shyamvyas5 — APPROVED** [review]

**Pull request merged**